include/uv-os390.h

```diff
@@ -26,5 +26,9 @@
 
 #define UV_PLATFORM_LOOP_FIELDS                                               \
   void* ep;                                                                   \
+  int fs_event_msg_queue;                                                     \
+
+#define UV_PLATFORM_FS_EVENT_FIELDS                                           \
+  char rfis_rftok[8];                                                         \
```

Comment:
fs_events is disabled for z/OS until this PR.
So this change doesn't really break any past behavior, right?